docs/account/getUTXOS.md

```diff
@@ -0,0 +1,10 @@
+**Usage**: `account.getUTXOS(onyAvailable)`      
```

Comment:
Parameter section below uses "onlySpendable" (vs the "onlyAvailable" used here). Also, missing the "l" in only.